x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/SpanFlyout/DatabaseContext.tsx

```diff
@@ -55,8 +54,10 @@ export function DatabaseContext({ dbContext }: Props) {
   }
 
   return (
-    <div>
-      <HeaderXSmall>DB Statement</HeaderXSmall>
+    <Fragment>
+      <EuiText>
```

Comment:
There's also EuiTitle, not 100% sure which is best for which

Reply:
We can use the EuiTitle `size=xs` for these titles